examples/aggressive_gauss.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
```

Comment:
Interesting approach to use these scripts. Is this for Tilo  or for us?

If it's for Tilo, it's not obvious exactly what each script is for. Aggressive Gauss vs. Baseline Benchmark, etc.

That being said, I'm not sure these examples will serve much purpose for Tilo  anyway, so perhaps a simple statement in the README that mentions their existence is good enough.

Reply:
Agreed, I'll add comments in the readme and in each script to make clear what they do.